spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -28,6 +28,17 @@
       expect(json['team']).to be_nil
     end
 
+    it 'returns data for users with invalid steam ids' do
+      @user.steamid = '0:0:000'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.